### PR TITLE
[Core][Installer] Rework installer commands

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | |
| License         | MIT |

*Sylius* installer is a powerful tool, but since last few years it became a little bit messy. This PR is the first step to make it usable.

- [x] read all commands and make sure they work properly (especially checking and setting permissions)
- [x] take care about coding standards
- [x] make output as minimal as it can be
- [x] extract commands logic to services to make them cleaner and more readable - the goal is to make command run service, optionally display start and end message, maybe run another command

To-do:
- [x] clean up ``SetupCommand``

I hope we will be able to test this commands properly in the nearest future, as current test doesn't really test anything ;)

### CODE REVIEW

**Comment by Zales0123** [comment]
UPDATE:
Most of the work is already done (and Installer still works 😄 ). I have some ideas how to make things better (especially - `SetupCommand` has one more big part to refactor), but this is a good part to be merged and be base for further Installer refactoring.

File: src/Sylius/Bundle/CoreBundle/Installer/Setup/LocaleSetup.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+     * {@inheritdoc}
+     */
+    public function setup(InputInterface $input, OutputInterface $output)
+    {
+        $name = $this->getLanguageName($this->locale);
+
+        $output->writeln(sprintf('Adding <info>%s</info> locale.', $name));
+
+        $existingLocale = $this->localeRepository->findOneBy(['code' => $this->locale]);
+        if (null !== $existingLocale) {
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Installer/Setup/LocaleSetup.php]: We should make sure it's enabled `$existingLocale->enable()`

File: src/Sylius/Bundle/CoreBundle/Installer/Setup/ChannelSetup.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        $this->channelManager = $channelManager;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setup(LocaleInterface $locale, CurrencyInterface $currency)
+    {
+        /** @var ChannelInterface $channel */
+        $channel = $this->channelRepository->findOneBy([]);
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Installer/Setup/ChannelSetup.php]: `$channel = $repo->findOneBy(['code' => 'default]);`

Comment (michalmarcinkowski) [inline src/Sylius/Bundle/CoreBundle/Installer/Setup/ChannelSetup.php]: `$channel = $repo->findOneBy(['code' => 'default]);`

**Comment by pjedrzejewski** [comment]
Thank you Mateusz, great work!

**Pull request merged**